gazebo_ros/scripts/debug

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 final="$@"
 
+# remove ros remapping arguments like "__name:=node_name"; otherwise, they'd 
+# be interpreted as a world name by gzserver, and it'd crash while loading it
+final=`echo "$final" | sed 's/[^ ]*:=[^ ]* \?//g'`
```

Comment:
Yes, it works. But I haven't known something like `<<<` exists. Isn't it more readable to do it with the `echo` call?